script/upload-to-github.js

```diff
@@ -13,9 +13,22 @@ let githubOpts = {
   filePath: filePath,
   name: fileName
 }
-github.repos.uploadAsset(githubOpts).then(() => {
-  process.exit()
-}).catch((err) => {
-  console.log(`Error uploading ${fileName} to GitHub:`, err)
-  process.exitCode = 1
-})
+
+let retry = true
+
+function uploadToGitHub () {
+  github.repos.uploadAsset(githubOpts).then(() => {
+    process.exit()
```

Comment:
Should we also include happy path output here? i.e. `Successfully uploaded <FILE> to GitHub`

Also wondering if two tries is enough.. any harm in doing more retries?

Reply:
@zeke yes it would be a good idea to include a success message - I'll add that.  
In regards to number of tries the reason I chose two was it appears to me that most times it is some sort of hiccup that goes away when we rerun the build.  The only downside in trying more times would be the jobs would run longer but I guess there wouldn't be too much time churned if we bumped up that number to say 5 times.